Dockerfiles/ci/command.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+OPERATOR_DIR="${OPERATOR_DIR:-/project/operator_dir/}"
+OPERATOR_WORK_DIR="${OPERATOR_WORK_DIR:-/project/test_operator_work_dir/}"
+WORK_DIR="${WORK_DIR:-/project/output/}"
+
+# prepares operator metadata for a give operator bundle based on 
+# $OPERATOR_DIR environment variable
+prepare_operator_metadata () {
+    ansible-playbook -vvv -i localhost, --connection local operator-test-playbooks/prepare-operator-metadata.yml -e "operator_dir=$OPERATOR_DIR" -e "operator_work_dir=$OPERATOR_WORK_DIR" -e"work_dir=$WORK_DIR"
+    cp /project/operator-test-playbooks/parse_operator_metadata_results.json $WORK_DIR
+    test -f $WORK_DIR/parsed_operator_data.yml
+    test -f $WORK_DIR/parse_operator_metadata_results.json
```

Comment:
We should fail if files do not exist.
We wilt just ignore failure with the current implementation

Reply:
@lslebodn In recent commit, I have made use of the python unit test framework as CMD inside Dockerfile. 
I felt it would be much easier for us to run/write unit tests. 
Please review. 
If approved I can squash both the commits.

Reply:
Using python unitest looks much better.